### DISCUSSION TITLE
TextLog: VERBOSE: Bouncer / OnNewProjectile

### DISCUSSION THREAD

**Discussion by nocturnalfilth** [Pryaxis/TShock#2105] [Q&A] [answered]
Hi, 

I am not very familiar with Terraria, but I set up a server for my kid. I installed the latest terraria server on linux and installed mono to get tshock working with the server for easier administration. however, I keep getting the message:
` TextLog: VERBOSE: Bouncer / OnNewProjectile please report to tshock about this! normally this is a reject from (and some player name).`

I have no idea what this means, but it's spamming the log. Is there anything I can do to fix this? Thanks for any help you can provide.

I am using 
- TShock4.4.0_Pre12_Terraria1.4.0.5 
- Terraria server 1405
- Ubuntu 20.04 LTS
- Latest version from Mono using their (https://download.mono-project.com/repo/ubuntu stable-focal main) repo.

**Comment by hakusaro** [comment, marked as answer]
You can turn debug logs off in the config file, but you can also use the vanillifier plugin if you want a vanilla terraria experience and you trust all of the users on the server.

**Reply by AndrewSav** [reply]
But what does that message mean? It says "please report to tshock about this", it sounds important.

**Reply by bartico6** [reply]
Debug logs are enabled by default on certain new Terraria version prereleases when we're not sure about certain mechanics being done right and let people who sign up for the prerelease test it. Those messages instruct you to report this to us **if this impacts gameplay** - if you can't shoot some gun legitimately and this message appears, you're supposed to tell us. There's always some small amount of false positives that don't really change anything and for those cases you just disable the logs.

**Reply by Twisted-Code** [reply]
Might want to be more specific with the string then? You seem to have a few people that similarly thought they should report it without regard to whether it affected gameplay... and if what's shown above is the exact string, I'd have thought the same, admittedly
Note: not to be taken as a complaint or anything. Just don't want you guys getting bogged down if people can't figure out how to use search to find previous threads like this...

**Reply by bartico6** [reply]
It sounds like a good call to maybe rephrase them, those were written hastily while updating and people paid more attention to making sure it triggers properly rather than the detailed content (they're **debug** logs after all). CC @QuiCM & @hakusaro on rephrasing them.

**Reply by hakusaro** [reply]
@macks2008 I assure you that the only thing we get bogged down by is discussion about the logs, not the actual logs

**Reply by AndrewSav** [reply]
@hakusaro rephrasing should certainly help with that. If it's more clear to people reading the message, the will be less questions and thus less discussion.

**Comment by moisterrific** [comment]
GitHub won't let me mark a reply to a potential answer as the answer, so I'm going to mark the answer that contains the reply from bartico6 that solves this question.
